src/llv8.cc

```diff
@@ -1569,6 +1757,17 @@ std::string JSObject::InspectDescriptors(Map map, Error& err) {
       continue;
     }
 
+    // Skip non-fields for now
```

Comment:
Remove this code as per https://github.com/nodejs/llnode/pull/221/files#diff-86a9160147028a03800f8ba4fa555285L1530